test/e2e/support/focus-commands.js

```diff
@@ -0,0 +1,34 @@
+Cypress.Commands.add( 'auditBlockFocus', ( blockType ) => {
+	// Open up the Insert Block Menu
+	cy.get( 'button.editor-inserter__toggle:first' ).click()
+	// Type into the Search field for blocks
+	cy.focused().type( blockType );
+	// Click on the first result that matches. Note, be careful with things like Image vs Cover Image
+	cy.get( '.editor-inserter__menu .editor-inserter__block:contains("' + blockType + '"):first' ).click();
```

Comment:
Re selectors, Is it a good time to introduce data-test attributes? or at least using constants to avoid repeating the selectors over and over again across tests. For example this could be: `FIRST_INSERTER_BLOCK`

Reply:
I'm glad we are on the same page. I really need to start reading all comments before I comment anything :)